### PR TITLE
works better with projectile

### PR DESCRIPTION
**Pull request by laishulu**
just enable the group by default.

### CODE REVIEW

File: modules/ui/tabs/config.el [hunk 1]
```diff
@@ -13,6 +13,7 @@
         ;; replaces the tab list with that of another Doom workspace. This
         ;; prevents that.
         centaur-tabs-cycle-scope 'tabs)
+  (centaur-tabs-group-by-projectile-project)
```
Comment (hlissner) [inline modules/ui/tabs/config.el:16]: Please move this after the `:config`, otherwise this breaks lazy loading of the package.

Comment (laishulu) [reply modules/ui/tabs/config.el]: pushed.

**Comment by hlissner** [comment]
Actually, on second thought, I'd rather we defer to centaur-tabs default on this one. Grouping tabs by project is an opinionated, rather than sensible default. At most, tabs should be grouped by perspectives (`persp-mode`) if `:ui workspaces` is enabled. Otherwise, I think this should be left to the user to add this to their own config.

```elisp
(after! centaur-tabs
  (centaur-tabs-group-by-projectle-project))
;; or
(add-hook 'centaur-tabs-mode-hook #'centaur-tabs-group-by-projectle-project)
```

In any case, thanks for the PR, but I must turn it down.

**Comment by laishulu** [comment]
Got it.
Totally understand your explaination.

**Pull request closed without merging**